commands/inspect.go

```diff
@@ -86,7 +86,7 @@ func runInspect(dockerCli command.Cli, in inspectOptions) error {
 					fmt.Fprintf(w, "Flags:\t%s\n", strings.Join(n.Flags, " "))
 				}
 				if nodes[i].Version != "" {
-					fmt.Fprintf(w, "Buildkit:\t%s\n", nodes[i].Version)
+					fmt.Fprintf(w, "BuildKit version:\t%s\n", nodes[i].Version)
```

Comment:
Didn't pick up on this before sorry! :see_no_evil: What's the reason for changing the field names (both here, and in `ls`)?

Reply:
I find this label confusing but let me remove this one from this PR.